### DISCUSSION TITLE
欢迎参与 CloudBase CMS 讨论

### DISCUSSION THREAD

**Discussion by chhpt** [TencentCloudBase/cloudbase-extension-cms#60] [General]
## 👋 Welcome!

欢迎使用 CloudBase CMS，任何关于 CloudBase CMS 的问题都可以在这里讨论~

在提出问题前，请先阅读[使用文档](https://docs.cloudbase.net/cms/intro.html)，避免无效问题。

**Comment by YuDe95** [comment]
微信群满了，还有什么方式进去吗
